tools/PAMPA/PAMPA_PLOT_GLM.xml

```diff
@@ -0,0 +1,143 @@
+<tool id="pampa_plotglm" name="Create a plot from GLM data" version="@VERSION@">
+    <description>as temporal trend</description>
+    <macros>
+        <import>pampa_macros.xml</import>
+    </macros>
+    <expand macro="Plot_requirements"/>
+    <command detect_errors="exit_code"><![CDATA[
+        Rscript 
+         '$__tool_directory__/FunctExePlotGLMGalaxy.r' 
+         '$input'
+         '$datatable'
+         '$unitobstable'
+         '$__tool_directory__/FunctPAMPAGalaxy.r' 
+         '$report'
+         '$plots'
+    ]]>
+    </command>
+    <inputs>
+        <param name="input" type="data" format="tabular" label="Input glm results file" help=""/>
+        <param name="datatable" type="data" format="tabular" label="Data table used for glm" help=""/>
+        <param name="unitobstable" type="data" format="tabular" label="Unitobs table used for glm" help=""/>
+    </inputs>
+    <outputs>
+       <data name="report" from_work_dir="stdout.txt" label="Report" format="txt"/>
+       <collection type="list" name="plots">
+            <discover_datasets pattern="(?P&lt;designation&gt;.+)\.png" visible="false" format="png"/>
+       </collection>
+    </outputs>
+    <tests>
+        <test> 

```

Comment:
can you add `expected_num_outputs` (https://docs.galaxyproject.org/en/latest/dev/schema.html#id78) here and maybe also in other tests?

Reply:
Thank you very much ! I'll do that